scripts/home.js

```diff
@@ -13,21 +10,33 @@ $.urlParam = function (a) {
 $(document).ready( ()=> {
     const name = $.urlParam("name");
     $("#name").text(name);
+    let featureDropdown = $("#feature-dropdown");
+    let dropdownMenuButton = $("#dropdownMenuButton");
+    let defaultDropDown = dropdownMenuButton.text();
 
-    post_to_annotation = function() {
-        let feature = $.urlParam("feature");
-        $.post("insert_name", {name: name})
-            .done(() => {
-                window.location.href = `annotation?batchid=1&name=${name}&feature=${feature}`;
-            })
-            .fail(() => {
-                // what would be the fail condition here?
-                alert(`Name is already in use, continuing as "${name}"`);
-                window.location.href = `annotation?batchid=1&name=${name}&feature=${feature}`;
-            });
-    };
+    function changeDropdownText(event) {
+        let ftr = event.data.ftr;
+        dropdownMenuButton.text(ftr);
+    }
 
-    post_to_admin = function() {
+    function postToAnnotation() {
+        let ftr = dropdownMenuButton.text();
+        if ( ftr === defaultDropDown) {
+            alert("Feature not selected from dropdown list.");
+        } else {
+            $.post("insert_name", {name: name})
+                .done(() => {
+                    window.location.href = `annotation?index=1&name=${name}&feature=${ftr}`;
```

Comment:
This should be `batchid` not `index`, and it will also have to get populated from the selection once that is implemented